snoop/data/urls.py

```diff
@@ -23,4 +23,5 @@
     path('<collection>/<hash>/raw/<filename>', views.document_download),
 
     path('<collection>/<hash>/tags/<username>/<uuid>', include(tags_router.urls)),
+    path('<collection>/thumbnail', views.thumbnail)
```

Comment:
make the URL look like the other ones:
- use the `<hash>` parameter in the URL instead of a `get` parameter called `digest`
- put the size in the URL too

Generally, it's really helpful to have the extension in the URL too (even if not required).

So make the URL look something like this instead:
`<collection>/<hash>/thumbnail/100.jpg`